# Jackpot — a quantitative teardown 🔬
### The textbook hedge with a Lo t-stat · the strengthening inversion · micro-caps, survivorship, regime

![Signal: None](https://img.shields.io/badge/Signal-None-c0392b?style=flat-square)
![Tradability: Mirage](https://img.shields.io/badge/Tradability-Mirage-c0392b?style=flat-square)
![Lottery effect on large caps?: Busted](https://img.shields.io/badge/Lottery_effect_on_large_caps%3F-Busted-8b949e?style=flat-square)

The deep companion to the [notebook for the curious](01_for_the_curious.ipynb). We show the lottery trade inverts on tradable large caps and why.

> ⚠️ **Not investment advice.** 436 S&P 500 names, daily, 2000–2026 (Yahoo); survivorship-biased, large-cap. Sources in [`docs/references.md`](../docs/references.md).

In [1]:
import sys, os
sys.path.insert(0, os.path.abspath("../../.."))  # repo root (quantlab/)
sys.path.insert(0, os.path.abspath(".."))        # study package (jackpot/)
%matplotlib inline
import matplotlib.pyplot as plt
plt.rcParams["figure.figsize"] = (10, 5.5); plt.rcParams["axes.grid"] = True
import numpy as np, pandas as pd
from jackpot import data, strategy as st
daily = data.fetch_panel()                     # cache-first
sig = st.max_signal(daily)
textbook = st.cross_section_hedge(daily, sig, long_high=False)   # long low-MAX, short high-MAX


## Verdict, up front

| Axis | Stamp | Why |
|---|---|---|
| Signal | **None** | textbook trade −10.4%/yr, Sharpe −0.49, t −2.5 |
| Tradability | **Mirage** | negative gross, inversion strengthening |
| Large-cap lottery? | **Busted** | premium lives in micro-caps |

> 💡 *In plain words:* the high-risk names won; the anomaly is a small-cap effect.

## 1 · The claim, steelmanned

- **H₁:** the long-low/short-high MAX trade earns a positive premium.
- **H₂:** it's stable.
- **H₃:** it survives on tradable large caps.

## 2 · So what? — what rides on each

If H₁/H₃ hold, you're paid to avoid lottery stocks anywhere. If they fail on large caps, the premium is a micro-cap, survivorship-fragile effect.

## 3 · How we'd know — the protocol

MAX each month → long-low/short-high → Lo t-stat → decade split.

## 4 · The teardown

### 4.1 The textbook trade

In [2]:
import pandas as pd
display(pd.Series(st.stats(textbook)).round(3))

mean_ann     -0.104
sharpe       -0.485
tstat        -2.524
hit_rate      0.448
n           328.000
dtype: float64

> 💡 *In plain words:* negative and significant (t −2.5). **H₁ rejected** — it inverted.

### 4.2 The inversion strengthened

In [3]:
for lab,sl in [('2000-2012',textbook.loc[:'2012']),('2013-on',textbook.loc['2013':])]:
    print(f'{lab}: Sharpe {st.stats(sl)["sharpe"]:+.2f}')

2000-2012: Sharpe -0.40
2013-on: Sharpe -0.71


> 💡 *In plain words:* −0.40 → −0.71 as the high-vol growth survivors compounded. **H₂ rejected.**

### 4.3 Why — three reasons

1. **MAX is a high-risk proxy** (vol/beta); on large caps 2000–2026 high risk *won* (mirror of [43 Free-Lunch](../../43-free-lunch/)).
2. **Survivorship**: the high-MAX blow-ups that delisted are gone; the high-MAX *survivors* did well.
3. **It's a micro-cap effect** (Hou-Xue-Zhang 2020) — the untradable end. **H₃ rejected.**

## 5 · The verdict

H₁, H₂, H₃ all rejected → Signal `NONE`, Tradability `MIRAGE`, large-cap lottery `BUSTED`.

## 6 · Could you trade it?

Not on liquid names. The premium is real in small/illiquid stocks, where shorting is costly and survivorship is worst — exactly where backtests flatter.

## 7 · Going further

Forks: (a) a point-in-time micro-cap universe (where the effect should reappear); (b) value-weighting; (c) MAX vs idio-vol head-to-head — see [Study 54 Static](../../54-static/). Backlog: [`docs/pwb_strategies_inventory.md`](../../../docs/pwb_strategies_inventory.md).